cinder/volume/drivers/lightos.py

```diff
@@ -1438,6 +1488,7 @@ def initialize_connection(self, volume, connector):
     def terminate_connection(self, volume, connector, **kwargs):
         force = 'force' in kwargs
         hostnqn = connector.get('nqn') if connector else None
+        host_ips = connector.get('host_ips',[]) 
```

Comment:
note the `if connector else None` above -- you can be called with a None connector

Reply:
If hostnqn is assigned value None, this code is not changing that behavior.  but host_ip will be empty list

Reply:
Doesn't matter, you will crash on a None connector when calling `connector.get`. You can only call `connector.get` if you have a non-None connector.

Reply:
got you, i added check for this